### ISSUE TITLE
Can not add Variants when defaultVariants is empty

### ISSUE THREAD

**Issue by davidhellmann** [nystudio107/craft3-imageoptimize#25]
I don't need default Variants cause I create three fields (square, landscape, portrait). But when I remove the default I can't add Variants on backend side. 
```
        // Default image variants
        'defaultVariants' => [
            /*[
                'width'          => 100,
                'useAspectRatio' => false,
                'aspectRatioX'   => 1.0,
                'aspectRatioY'   => 1.0,
                'retinaSizes'    => ['1'],
                'quality'        => 82,
                'format'         => 'jpg',
            ],*/
        ],
```

![image](https://user-images.githubusercontent.com/764292/34019564-e7216920-e12f-11e7-84c3-d1a4f8d06b88.png)

**Comment by khalwat** [comment]
This is being "fixed" in 1.4.3, in that it will require that you have at least 1 default variant

**Comment by khalwat** [comment]
This is fixed in 1.4.3
